bitmap-team0/lib/TransformableBitmap.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = TransformableBitmap;
+
+function TransformableBitmap(path) {
+  this.bitmap = fs.readFileSync(path);
+  this.colorTableOffset = 14 + this.bitmap.readUInt32LE(14);
+	//this.width = this.bitmap.readUInt32LE(18);
+	//this.height = this.bitmap.readUInt32LE(22);
```

Comment:
@bazacko @bhavyaab @ShaAllFar best practice is to remove these comment blocks from the `master` branch